src/components/Form/BugInForm.js.example

```diff
@@ -0,0 +1,2 @@
+// Rename this file from BugInForm.js.example to BugInForm.js to "activate" the demo so react-styleguidist can see it
+export function BugInForm() {}
```

Comment:
These `BugInForm.*.example` files are meant to be a convenient template where you can quickly cut a but-topic-branch and push  up an example of the issue.